docs/handlers.md

```diff
@@ -0,0 +1,1584 @@
+| Format        | Type                                 | Fully supported?    |
+| :------------ | :----------------------------------- | :-----------------: |
+| [`chk`](#chk) | ARCHIVE | :octicons-check-16: |
+| [`trx_v1`](#trx_v1) | ARCHIVE | :octicons-check-16: |
+| [`trx_v2`](#trx_v2) | ARCHIVE | :octicons-check-16: |
+| [`encrpted_img`](#encrpted_img) | ARCHIVE | :octicons-check-16: |
+| [`shrs`](#shrs) | ARCHIVE | :octicons-check-16: |
```

Comment:
Some of these formats are vendor specific (e.g Netgear and D-Link above), while most are vendor neutral/generic (zip, tar, ar, iso9660, ...)

I miss this vendor information in the table.

Also, I would expect lexical sorting, but the table is unsorted.

Reply:
Two excellent points. I'll take care of it :)